.travis.yml

```diff
@@ -17,7 +18,24 @@ before_script:
   - composer selfupdate --quiet
   - composer install --prefer-dist --no-interaction --no-progress
   - ./vendor/bin/phpunit
-  - composer update --prefer-dist --no-interaction --no-progress
```

Comment:
Yes, I wasn't aware that the composer.lock was not there. 
Then the phpunit is begin executed twice as you said.